app/templates/public.hbs

```diff
@@ -38,7 +38,7 @@
       <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="three wide column public">
+      <div class="{{if device.isMobile 'external-link' 'three wide column public'}}">
```

Comment:
what's `external-link` ? by not leaving it empty, I meant if it's a three-wide column on a large screen, then there is a column size which is correct for mobile screen. Use those semantic UI classes.